source/features/remove-unused-repo-tabs.tsx

```diff
@@ -0,0 +1,105 @@
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import fetchDom from '../helpers/fetch-dom';
+import getTabCount from '../github-helpers/get-tab-count';
+import looseParseInt from '../helpers/loose-parse-int';
+import {getWorkflows} from './next-scheduled-github-action';
+import abbreviateNumber from '../helpers/abbreviate-number';
+import {onlyShowInDropdown} from './more-dropdown';
+import {buildRepoURL, getRepo} from '../github-helpers';
+
+function tabCannotBeHidden(tab: HTMLElement | undefined): boolean {
+	return (
+		!tab || // Tab disabled 🎉
+		tab.matches('.selected') ||// User is on tab 👀
+		// Repo/Organization owners should see the tab. If they don't need it, they should disable the feature altogether
+		pageDetect.canUserEditRepo() ||
+		pageDetect.canUserEditOrganization()
+	);
+}
+
+function setTabCounter(tab: HTMLElement, count: number): void {
+	const tabCounter = select('.Counter', tab)!;
+	tabCounter.textContent = abbreviateNumber(count);
+	tabCounter.title = count > 999 ? String(count) : '';
+}
+
+const getWikiPageCount = cache.function(async (): Promise<number> => {
+	const wikiPages = await fetchDom(buildRepoURL('wiki'), '#wiki-pages-box .Counter');
+	return looseParseInt(wikiPages);
+}, {
+	maxAge: {hours: 1},
+	staleWhileRevalidate: {days: 5},
+	cacheKey: () => 'wiki-page-count:' + getRepo()!.nameWithOwner
+});
+
+async function initWiki(): Promise<void | false> {
+	const wikiTab = await elementReady('[data-hotkey="g w"]');
+	if (!wikiTab) {
+		return false;
+	}
+
+	const wikiPageCount = await getWikiPageCount();
+	if (tabCannotBeHidden(wikiTab) || wikiPageCount > 0) {
```

Comment:
`> 0` is faster on my eyes and on the CPU
```suggestion
	if (wikiPageCount > 0 || tabCannotBeHidden(wikiTab)) {
```